README.md

```diff
@@ -59,4 +59,4 @@ Instructions
    the template at the best fit redshift.  This may need to be 
    repeated for different redshifts
 
-
+10. run agnextract to produce the spectrum
```

Comment:
Can you add this as a different section?  This is separate from those above.  So something named 

    Instructions for AGN Extraction  

And then add a few more details like an example of running the code